charts/trino/templates/deployment-worker.yaml

```diff
@@ -44,6 +44,9 @@ spec:
         runAsUser: {{ .runAsUser }}
         runAsGroup: {{ .runAsGroup }}
       {{- end }}
+      {{- if .Values.shareProcessNamespace.worker }}
```

Comment:
why conditionally?

Reply:
To not force user to define `shareProcessNamespace`

Reply:
isn't default always a fallback anyway?

Reply:
and you would just put `shareProcessNamespace: false` if it is not redefined explicitly?

Reply:
changed.